dap_prinz_green_jobs/pipeline/green_measures/industries/sic_mapper/sic_mapper_utils.py

```diff
@@ -108,37 +139,35 @@ def clean_company_name(
         return None
 
 
-# We can modify this slightly when we use a company name to SIC mapper
-# instead of companies house data
-def get_ch_sic(company_name: str, ojo_companies_house_dict: Dict) -> str:
-    """
-    Pick the first 5 digit SIC for each cleaned name
-        using the Companies House data. Assumes that companies house
-        SICs are ordered in terms of importance.
+def clean_company_description(
+    description: str, bad_phrases: List[str] = bad_phrases
+) -> str:
+    """Minimal cleaning of company description.
 
-    :type companies_house_cleaned_in_ojo_dict: dict
+    Args:
+        description (str): The company description
 
-    :param cleaned_name: The cleaned company name
-    :type cleaned_name: str
+    Returns:
+        str: The cleaned company description
+    """
+    sentence_replacement_rules = {
+        r"\b(?:"
+        + "|".join(map(re.escape, bad_phrases))
+        + r")\b": "",  # Remove bad phrases
```

Comment:
my regex isn't good enough, but based off my comment about bad_phrases, is this removing them only if they are at the beginning of the description then? if so, `# Remove bad phrases` -> `# Remove bad phrases if at the beginning of the description`